Atomic/util.py

```diff
@@ -254,3 +253,19 @@ def urllib3_disable_warnings():
             # Except only call disable-warnings if it exists
             if hasattr(urllib3, 'disable_warnings'):
                 urllib3.disable_warnings()
+
+
+def skopeo(image):
```

Comment:
@runcom  In speaking with @rhatdan , we decided we would replace remote inspection with one PR and then fix verify afterwards.  So I am waiting for this PR to merge before fixing verify wrt images and fully qualified images.